Makefile

```diff
@@ -1,20 +1,24 @@
 default: build
 
-lint: vendor
+PACKAGES=./libs/... ./internal/... ./cmd/... ./bundle/... .
+
+lint:
 	@echo "✓ Linting source code with https://golangci-lint.run/ (with --fix)..."
 	@./lint.sh ./...
 
-lintcheck: vendor
+lintcheck:
 	@echo "✓ Linting source code with https://golangci-lint.run/ ..."
 	@golangci-lint run ./...
 
-test: lint testonly
-
-testonly:
+test:
 	@echo "✓ Running tests ..."
-	@gotestsum --format pkgname-and-test-fails --no-summary=skipped --raw-command go test -v -json -short -coverprofile=coverage.txt ./...
+	@gotestsum --format pkgname-and-test-fails --no-summary=skipped -- ${PACKAGES}
```

Comment:
It seems like `-short` test is skipped, is it intentional?

Reply:
Yes, it's intentional but I forgot to mention it. We do not use short/non-short in unit tests, so no reason to pass this flag.